packages/data/readme.md

```diff
@@ -1,3 +1,83 @@
 # `@directus/data`
 
-Data abstraction for Directus
+> **Warning** This is a work in progress. Nothing is expected to work yet.
+
+## Installation
+
+```
+npm install @directus/data
+```
+
+## Usage
+
+Create a new manager:
+
+```js
+import { DataEngine } from '@directus/data';
+
+const data = new DataEngine();
+```
+
+Register drivers:
+
+```js
+import { DataEngine } from '@directus/data';
+import { DataDriverPostgres } from '@directus/data-driver-postgres';
+
+const data = new DataEngine();
+
+data.registerStore(
+	'postgres',
+	new DataDriverPostgres({
+		connectionString: 'postgresql://root:password@localhost/mydb',
+	})
+);
+```
```

Comment:
I wonder if the registration logic should be done within the data package itself instead within the `api`. This would also include the logic of dynamically loading the actual driver. In this constellation the `api` would only have a dependency to `data` which in turn would have dependencies to all the drivers, instead of the `api` having dependencies  to `data` _and_ all the drivers directly.

The `api` would then only need to figure out which driver to use and pass the string to the `data` package. it feels like the handling and registration of drivers is more a think for `data` instead of for `api` 🤔